### DISCUSSION TITLE
After installing Open WebUI, visiting `http://127.0.0.1:42000` returns {"detail": "Not Found"}

### DISCUSSION THREAD

**Discussion by wjf68888** [open-webui/open-webui#5552] [General]
After installing Open WebUI, visiting `http://127.0.0.1:42000` returns {"detail": "Not Found"}, while visiting
OLLAMA at `http://127.0.0.1:11434` displays “Ollama is running”. This suggests that Open WebUI might not be
installed or configured correctly. However, I am unsure of what to do next and could use some help.
![微信图片_20240920145843](https://github.com/user-attachments/assets/15e6e75a-a442-47ec-acf2-279b1266bc35)
![微信图片_20240920145843](https://github.com/user-attachments/assets/541ceb81-07b9-4d60-96b3-df999b5d7ff5)

**Comment by aadlung** [comment]
I assume you found out that you need to run "npm i" and "npm run build" on the frontend as well...
